### PR TITLE
Development of relationship model, service and router

### PR DESCRIPTION
**Pull request by paulinaborys**

### CODE REVIEW

File: graph_api/relationship/relationship_model.py [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+            end_node (Optional[int]): Id of node which ends connection
+            name (Optional[str]): Name of the relationship
     """
 
-    start_node: int
-    end_node: int
-    name: str
+    start_node: Optional[int]
+    end_node: Optional[int]
+    name: Optional[str]
```
Comment (RioluM) [inline graph_api/relationship/relationship_model.py:18]: I'm not sure if this should be Optional

File: graph_api/relationship/relationship_router.py [hunk 2]
```diff
@@ -32,3 +34,18 @@ async def create_relationship(self, relationship: RelationshipIn, response: Resp
         create_response.links = get_links(router)
 
         return create_response
+
+    @router.post("/relationships/{id}/properties", tags=["relationships"], response_model=RelationshipOut)
+    async def create_relationship_properties(self, id: int, properties: List[PropertyIn], response: Response):
+        """
+        Create properties for relationship with given id
+        """
+        print(id)
```
Comment (RioluM) [inline graph_api/relationship/relationship_router.py:43]: Some random print :P

File: graph_api/relationship/relationship_model.py (hunk 1 above)
Comment (mrwrob) [reply graph_api/relationship/relationship_model.py:18]: I also cannot think of a case where there is a relationship without a start or end node. Perhaps there can be a relationship without a name.... @terzawad?

File: graph_api/relationship/relationship_service.py [hunk 3]
```diff
@@ -70,3 +92,38 @@ def save_relationship(self, relationship: RelationshipIn):
...
+                                     auth=self.database_auth).json()
+            if len(response["errors"]) > 0:
+                result = RelationshipOut(errors=response["errors"])
+            else:
+                relationship_id = response["results"][0]["data"][0]["meta"][0]["id"]
+                result = RelationshipOut(id=relationship_id, properties=properties)
+        else:
+            result = RelationshipOut(id=id, errors={"errors": "not matching id"})
+
+        return result
```
Comment (mrwrob) [inline graph_api/relationship/relationship_service.py:129]: According to PEP-8 there should be newline at the end of the file: https://pep8.org/

**Review by mrwrob — CHANGES_REQUESTED** [review]

File: graph_api/relationship/relationship_model.py (hunk 1 above)
Comment (mrwrob) [reply graph_api/relationship/relationship_model.py:18]: For now, lets remove Optional from `start_node` and `end_node` and leave it for `name`.

**Comment by mrwrob** [comment]
We need this feature for Sprint Review, so I am merging this PR. However, we need to remember to check this _Optional_ issue.

**Pull request merged**